### PR TITLE
Tooltip not displaying on coverage icon if an event/planning item was created using 'duplicate' option [CPCN-899]

### PR DESCRIPTION
**Pull request by devketanpro**
add coverage assignee details and coverage status on duplicate planning

## Front-end checklist

- [ ] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [ ] This pull request is using TypeScript interfaces instead of prop-types and updates usages where it's quick to do so
- [ ] This pull request is using `memo` or `PureComponent` to define new React components (and updates existing usages in modified code segments)
- [ ] This pull request is replacing `lodash.get` with optional chaining and nullish coalescing for modified code segments
- [ ] This pull request is not importing anything from client-core directly (use `superdeskApi`)
- [ ] This pull request is importing UI components from `superdesk-ui-framework` and `superdeskApi` when possible instead of using ones defined in this repository.
- [ ] This pull request is not using `planningApi` where it is possible to use `superdeskApi`
- [ ] This pull request is not adding redux based modals
- [ ] In this pull request, properties of redux state are not being passed as props to components; instead, we connect it to the component that needs them. Except components where using a react key is required - do not connect those due to performance reasons.
- [ ] This pull request is not adding redux actions that do not modify state (e.g. only calling angular services; those should be moved to `planningApi`)

### CODE REVIEW

**Comment by devketanpro** [comment]
Hey @MarkLark86, do we need to add a config for this?

**Comment by MarkLark86** [comment]
> Hey @MarkLark86, do we need to add a config for this?

Yes, and probably default to the current behaviour. Also add it to the config section in the README.md file.

File: server/planning/planning/planning_duplicate.py [hunk 1]
```diff
@@ -99,12 +99,11 @@ def _duplicate_planning(self, original):
...
 
         for cov in new_plan.get("coverages") or []:
-            cov.pop("assigned_to", None)
             cov.get("planning", {}).pop("workflow_status_reason", None)
             cov.pop("scheduled_updates", None)
             cov.get("planning", {})["scheduled"] = new_plan.get("planning_date")
             cov["coverage_id"] = TEMP_ID_PREFIX + "duplicate"
             cov["workflow_status"] = WORKFLOW_STATE.DRAFT
-            cov["news_coverage_status"] = {"qcode": "ncostat:int"}
+            cov["news_coverage_status"] = {"qcode": "ncostat:int", "label": "Planned", "name": "coverage intended"}
```
Comment (MarkLark86) [inline server/planning/planning/planning_duplicate.py]: Ideally this should come from the CV

**Review by MarkLark86 — APPROVED** [review]
There are some tests failing. You might be able to fix those behave tests by adding `@planning_cvs`, which will add the CV required

**Pull request merged**